test/cmd/txsim/cli.go

```diff
@@ -2,8 +2,10 @@ package main
 
 import (
 	"context"
+	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/celestiaorg/go-square/v2/share"
```

Comment:
golangci-lint is complaining.

TBH I dislike this lint rule so I would also be okay with disabling the custom settings [here](https://github.com/celestiaorg/celestia-app/blob/ea39814e10c87ae883b8eedce0a020d59060ef9f/.golangci.yml#L44-L50).